src/main/scala/MainClass/StudentDAO.scala

```diff
@@ -78,19 +78,19 @@ object postGreSQLConnect {
     val a=new StudentDAO
     val b=a.getConnection()
     val c=a.createStatement(b)
-//a.createTable(c)
-//    val s1= Student(1,"Aman","History",89)
-//    val s2= Student(2,"Alex","Maths",70)
-//    val s3= Student(3,"Alok","English",56)
-//    val s4= Student(4,"Ankit","Computers",80)
-//    val s5=Student(5,"Akshata","Science",85)
-//    val s6=Student(8,"Daniel","Marketing",45)
-//    a.insert(c,s1)
-//    a.insert(c,s2)
-//    a.insert(c,s3)
-//    a.insert(c,s4)
-//    a.insert(c,s5)
-//    a.insert(c,s6)
+    a.createTable(c)
```

Comment:
Try to create a switch case kind of iplementation where you ask user to select the desired task whether they have to perform s select operation or insert etc...

Reply:
looks good now